dialogue-core/src/main/java/com/palantir/dialogue/core/WeakReducingGauge.java

```diff
@@ -32,14 +35,18 @@
  * the final summary integer.
  */
 @ThreadSafe
-final class WeakSummingGauge<T> implements Gauge<Integer> {
-    private final ToIntFunction<T> gaugeFunction;
+public final class WeakReducingGauge<T> implements Gauge<Number> {
```

Comment:
Had to make this public because it's needed in `com.palantir.dialogue.{hc4, core}`. Don't really think it's worth moving to tritium, but I also don't really want people to rely on it :/

Reply:
We can add `internal` to either the package or class name ¯\_(ツ)_/¯